contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -917,11 +713,13 @@ contract LoanTokenLogicStandard is AdvancedToken, StorageExtension {
                 _safeTransfer(collateralTokenAddress, bZxContract, collateralTokenSent, "28");
                 msgValue -= collateralTokenSent;
             } else {
+                loanDataBytes = _checkPermit(collateralTokenAddress, loanDataBytes);
```

Comment:
no need to write back to loanDataBytes here

Reply:
resolving